cpu/numactl.py

```diff
@@ -67,6 +72,57 @@ def setUp(self):
 
         build.make(self.sourcedir)
 
+        self.iface = self.params.get("interface", default="")
+        self.ping_count = self.params.get("ping_count", default=100)
+        self.peer = self.params.get("peer_ip", default="")
+        interfaces = netifaces.interfaces()
+        if not self.iface:
+            self.cancel("Please specify interface to be used")
+        if self.iface not in interfaces:
+            self.cancel("%s interface is not available" % self.iface)
+        if not self.peer:
+            self.cancel("peer ip should specify in input file")
```

Comment:
reword as peer ip need to specify in YAML

Reply:
@PraveenPenguin fixed, reworded as suggested.